netbox/extras/forms/__init__.py

```diff
@@ -4,5 +4,4 @@
 from .bulk_import import *
 from .misc import *
 from .mixins import *
-from .config import *
```

Comment:
We should either move ConfigRevisionForm into `model_forms.py` or leave it here, but not have two copies of it.